service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/OpenApiConfiguration.java

```diff
@@ -170,7 +171,14 @@ protected void addOpenApiSpecEndpoints(RouterFunctions.Builder routerFunctions,
     CoreRequestHandler coreRequestHandler =
         new CoreRequestHandler(openApi, httpMethodOperation.getName(), responseSchemaContext, responseContextValidator,
             graphQl, responseMapper, paramHandlerRouter, requestBodyHandlerRouter, jexlHelper, environmentProperties);
-    coreRequestHandler.validateSchema();
+
+    for (ResponseTemplate responseTemplate : responseTemplates) {
+      HttpStatus httpStatus = HttpStatus.valueOf(responseTemplate.getResponseCode());
+      if (!httpStatus.is3xxRedirection()) {
+        coreRequestHandler.validateSchema();
```

Comment:
Vreemd dat dit binnen de loop gebeurd zonder variabele uit de loop te gebruiken. Potentieel gebeurt dit nu ook meerdere keren.
Zoiets?
```suggestion
responseTemplates.stream().map(rt->HttpStatus.valueOf(rt.getResponseCode())).filter(httpStatus->!httpStatus.is3xxRedirection()).findFirst().ifPresent(i->coreRequestHandler.validateSchema());
```

Reply:
aangepast